src/View/Components/Hr.php

```diff
@@ -28,10 +28,10 @@ public function progressTarget(): ?string
     public function render(): View|Closure|string
     {
         return <<<'HTML'
-                <div class="h-[2px] border border-t-base-100 my-5">
+                <div class="h-[2px] border border-base-content/10 my-5">
                     <progress
-                        class="progress progress-primary hidden h-[1px]"
-                        wire:loading.class="!h-[2px] !block"
+                        class="progress progress-primary hidden -mt-[1px] h-[2px]"
+                        wire:loading.class="!block"
```

Comment:
On light theme this makes the line larger. It should looks like what was before.

Reply:
fixed, both now thinner again